docs/source/programming.rst

```diff
@@ -6,6 +6,8 @@ This page describes how to load compiled firmware onto an Ubertooth One. Other p
 
 There are three ways to program an Ubertooth board: the USB bootloader, the ISP bootloader, and JTAG.
 
+Please note that it is not possible to program the board's usb firmware via ubertooth-dfu as there is software protection in place. 
```

Comment:
Please change "program" to "replace" and change "usb firmware" to "USB bootloader".

Reply:
Done.